scipy/special/tests/test_precompute_gammainc.py

```diff
@@ -36,19 +40,21 @@ def test_g():
 @check_version(sympy, '0.7')
 def test_alpha():
     # Test data for the alpha_k. See DLMF 8.12.14.
+    rtol = 2e-11 if _is_32bit_platform else 1e-17
```

Comment:
Why does that help? This at least catches any further regressions.

I also don't think a warning is needed here - we have many tests that pass with tolerances of this order of magnitude.